syntax = "proto3";

import "api/datatype.proto";

package synapse;

message StreamInConfig {
  repeated uint32 shape = 1;
  DataType data_type = 2;
}
